Microsoft.Toolkit.Uwp.UI.Controls/InfiniteCanvas/InfiniteCanvas.xaml

```diff
@@ -96,13 +96,35 @@
                                         </ToggleButton.Content>
                                     </ToggleButton>
 
-                                    <TextBox x:Name="CanvasTextBoxFontSizeTextBox"
-                                             Width="64"
-                                             Height="32"
-                                             InputScope="Number"
-                                             MaxLength="3"
-                                             Text="22"
-                                             ToolTipService.ToolTip="Font Size" />
+
+                                    <ComboBox x:Name="CanvasComboBoxFontSizeTextBox"
```

Comment:
We should make sure we keep the tooltip/automation property name for narrator. Though we should set through the resources file.

Also, I think from the UX perspective, we'd need this to be editable with the `IsEditable` flag, doesn't seem as straight-forward to just enable that though, testing this out quick on a branch.

That also leads us to the discussion in #3440 as `IsEditable` is a 1809 feature and conditional XAML seems wonky here.

Reply:
Will add that back

Reply:
After #3214 you don't have to worry if `IsEditable` is avaible as we target 1809 and up now. You can just set it to true.